### PR TITLE
Same JMS examples proposal

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Dmitri Fedorov <dfedorov.solace@gmail.com>
Per conversation last week.

### CODE REVIEW

**Comment by ghost** [comment]
See corresponding changes in https://github.com/dfedorov-solace/solace-samples-amqp-jms1/pull/9

**Comment by mdspielman** [comment]
Jonathan should be able to review the submission tomorrow.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>solace-samples</groupId>
+    <artifactId>solace-samples-jms</artifactId>
+    <version>1.0.1-SNAPSHOT</version>
+    <name>Solace JMS 1.1 Examples</name>
+    <description>Solace Messaging Examples for JMS 1.1</description>
+    <dependencies>
+        <dependency>
+            <groupId>com.solacesystems</groupId>
+            <artifactId>sol-jms</artifactId>
```
Comment (jschabowsky) [inline pom.xml:12]: Why is this not using QPID? Looks to be using Solace JMS

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>solace-samples</groupId>
+    <artifactId>solace-samples-jms</artifactId>
+    <version>1.0.1-SNAPSHOT</version>
+    <name>Solace JMS 1.1 Examples</name>
```
Comment (jschabowsky) [inline pom.xml:7]: Rename this to be AMQP QPID JMS 1.1 Examples

File: src/main/java/com/solace/samples/TopicPublisher.java [hunk 3]
```diff
@@ -17,6 +17,10 @@
  * under the License.
  */
 
+/**
+ *  Solace JMS 1.1 Examples: TopicPublisher
```
Comment (jschabowsky) [inline src/main/java/com/solace/samples/TopicPublisher.java:21]: Naming. This should be QPID or Something... We need to be sure we are making a differentiation between Solace JMS and ApacheQpid JMS 1.1

File: src/main/java/com/solace/samples/TopicPublisher.java [hunk 4]
```diff
@@ -26,57 +30,70 @@
...
 
+/**
+ * Publishes a messages to a topic using Solace JMS 1.1 API implementation.
+ * 
+ * This is the Publisher in the Publish/Subscribe messaging pattern.
+ */
 public class TopicPublisher {
 
-    public void run(String... args) throws Exception {
+    final String SOLACE_HOST = "192.168.133.8:55555";
```
Comment (jschabowsky) [inline src/main/java/com/solace/samples/TopicPublisher.java:44]: This is not the AMQP port

File: src/main/java/com/solace/samples/TopicSubscriber.java [hunk 5]
```diff
@@ -17,6 +17,9 @@
  * under the License.
  */
 
+/**
+ *  Solace JMS 1.1 Examples: TopicSubscriber
```
Comment (jschabowsky) [inline src/main/java/com/solace/samples/TopicSubscriber.java:21]: Naming Again

File: src/main/java/com/solace/samples/TopicSubscriber.java [hunk 6]
```diff
@@ -34,79 +37,81 @@
 import com.solacesystems.jms.SolConnectionFactory;
 import com.solacesystems.jms.SolJmsUtility;
 
+/**
+ * Subscribes to messages published to a topic using Solace JMS 1.1 API implementation.
```
Comment (jschabowsky) [inline src/main/java/com/solace/samples/TopicSubscriber.java:41]: Apache Qpid 1.1 API

File: src/main/java/com/solace/samples/TopicSubscriber.java [hunk 7]
```diff
@@ -34,79 +37,81 @@
...
+ *
+ * This is the Subscriber in the Publish/Subscribe messaging pattern.
+ */
 public class TopicSubscriber {
 
-    public void run(String... args) throws Exception, JMSException, NamingException {
-        System.out.println("TopicSubscriber initializing...");
+    final String SOLACE_HOST = "192.168.133.8:55555";
```
Comment (jschabowsky) [inline src/main/java/com/solace/samples/TopicSubscriber.java:47]: AMQP Port?

**Review by jschabowsky — CHANGES_REQUESTED** [review]
Looks really good. 

See comments inline but also:
1) Why did you hard code HOST, VPN, Username/Password instead of having a properties file? 
2) This should not be merged into solace-samples-jms... It needs its own repo...  Talk to Mark S. but am thinking: Apache-QPID-JMS

File: pom.xml (hunk 1 above)
Comment (ghost) [reply pom.xml:12]: Because this the tutorial on using Solace JMS API implementation.

File: pom.xml (hunk 2 above)
Comment (dfedorov-solace) [inline pom.xml]: Why? This is the Solace JMS API implementation tutorial, not QPID

**Comment by ghost** [comment]
Jonathan, I'm cancelling this pull request to revert to the command line arguments for the Solace router hostname/port. Same change will be done in corresponding AMQP QPID JMS programs.

**Comment by mdspielman** [comment]
As per offline conversation, code is moving in the right direction towards mergeable. To finish this pull request please address the following:

* Markdown tutorial code snippets and sample code should be copies of each other. Example of where it is not is connection factory creation in TopicPublisher. 
* If choosing to make these style changes, please reflect them in all tutorials and associated markdown tutorials within the project. I'd like all tutorials to updated in one pull request to keep the style uniform.

**Pull request closed without merging**